test/server/cards/06-CotE/IkomaOrator.spec.js

```diff
@@ -0,0 +1,64 @@
+describe('Ikoma Orator', function() {
+    integration(function() {
+        describe('Ikoma Orator\'s ability', function() {
+            beforeEach(function() {
+                this.setupTest({
+                    phase: 'conflict',
+                    player1: {
+                        inPlay: ['ikoma-orator'],
+                        hand: ['assassination', 'banzai'],
+                        honor: 12
+                    },
+                    player2: {
+                        inPlay: ['bayushi-liar'],
+                        honor: 11
+                    }
+                });
+                this.ikomaOrator = this.player1.findCardByName('ikoma-orator');
+                this.bayushiLiar = this.player2.findCardByName('bayushi-liar');
+                this.shamefulDisplay = this.player2.findCardByName('shameful-display', 'province 1');
+                this.assassination = this.player1.clickCard('assassination');
+                this.banzai = this.player1.clickCard('banzai');
+
+                this.noMoreActions();
+
+                this.initiateConflict({
+                    type: 'political',
+                    province: this.shamefulDisplay,
```

Comment:
Not a problem doing this, but you do not usually need to specify the province with the `this.initiateConflict` function.  Just to let you know.

Reply:
Good to know, I removed the Shameful definition and this line as well.